tests/unit/repo/test_keys

```diff
@@ -0,0 +1 @@
+../../integration/repo/test_keys/
```

Comment:
we should probably move these `test_keys` to a single place with a fixture or something to access them (not on this PR though)

Reply:
Agreed. Especially as we get more integration tests it'll be good to have a common location for other data files (such as yaml snippets for archives)